Libraries/libcar/Headers/car/Writer.h

```diff
@@ -60,6 +67,17 @@ class Writer {
      */
     void addRendition(Rendition const &rendition);
 
+    /*
+     * Add a rendition for a facet, optimized for fast editing of CAR files
+     */
+    void addRendition(void *key, size_t key_len, void *value, size_t value_len);
+
+    /*
+     * The key format, optional and determined automatically if omitted.
+     */
+    ext::optional<struct car_key_format *> &keyfmt()
+    { return _keyfmt; }
```

Comment:
I don't see these used anywhere in this PR — are you using this to copy information from another car file?

The current architecture of separate `Reader` and `Writer` objects is probably not the best one if you're trying to make small modifications to existing files, but my understanding was that wasn't being done or likely needed. If that's going to be needed, it might be worth reconsidering the architecture of `libcar` so the internals aren't so widely exposed.